templates/blocks/block--system-branding-block.html.twig

```diff
@@ -6,12 +6,30 @@
  * @see ./core/modules/system/templates/block--system-branding-block.html.twig
  */
 #}
+{% set _header_link = {
+  url: url('<front>'),
+  aria_label: site_name
+}
+%}
+{% set _header_image = {
```

Comment:
Is there any reason to set this variable if we are never going to use it in full?